src/core/AutoRest.Core/Properties/Resources.resx

```diff
@@ -378,4 +378,7 @@ Modelers:
   <data name="TrackedResourcePatchOperationMissing" xml:space="preserve">
     <value>Tracked resource '{0}' must have patch operation that at least supports the update of tags.</value>
   </data>
+  <data name="PropertyNameShouldBeCamelCase" xml:space="preserve">
+    <value>Property names should follow camelCase style: {0}.</value>
```

Comment:
nit: For better clarity, can we make the message look like "Property name {0} should follow camelCase style: {1}." where 0 is the violating property and 1 is the suggested correct format?